app/menus/NewDocumentMenu.js

```diff
@@ -45,7 +44,7 @@ function NewDocumentMenu() {
       <MenuButton {...menu}>
         {(props) => (
           <Button icon={<PlusIcon />} {...props} small>
-            {`${t("New doc")}…`}
+            {`${t("New doc")}`}
```

Comment:
```suggestion
            {`${t("New doc")}…`}
```

This was actually purposeful to signify that there is another menu/choice